src/device/epoll.rs

```diff
@@ -132,7 +133,7 @@ impl<H: Sync + Send> EventPoll<H> {
 
         let ts = timespec {
             tv_sec: period.as_secs() as _,
-            tv_nsec: period.subsec_nanos() as _,
+            tv_nsec: i64::from(period.subsec_nanos()),
```

Comment:
Not portable (32-bit builds fail), but surely the old code was fine?

Reply:
Clippy was complaining